tests/core/builder-tools/test_chain_construction.py

```diff
@@ -82,7 +83,7 @@ def test_chain_builder_construct_chain_vm_configuration_multiple_forks():
         (constantinople_at, ConstantinopleVM),
         (petersburg_at, PetersburgVM),
         (istanbul_at, IstanbulVM),
-        (latest_mainnet_at, PetersburgVM),  # this will change whenever the next upgrade is locked
+        (latest_mainnet_at, MuirGlacierVM),  # this will change whenever the next upgrade is locked
```

Comment:
should this list also include a `(muir_glacier_at, MuirGlacierVM)`?

Reply:
It should :+1: